src/main/java/org/mongoflink/sink/MongoSink.java

```diff
@@ -30,37 +28,38 @@
 
     private final MongoClientProvider clientProvider;
 
-    private DocumentSerializer<IN> serializer;
+    private final DocumentSerializer<IN> serializer;
 
-    private final SinkConfiguration configuration;
+    private final MongoConnectorOptions options;
 
-    public MongoSink(String connectionString,
```

Comment:
As mentioned above, we'd better keep this constructor to be compatible.

Reply:
done